spec/support/test_server.rb

```diff
@@ -34,9 +34,11 @@ def find_available_port
   def call(env)
     @root = File.expand_path(File.dirname(__FILE__))
     path = Rack::Utils.unescape(env['PATH_INFO'])
-    case path
-    when '/get_success'
+    method = env['REQUEST_METHOD']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.